### ISSUE TITLE
there is one more issue, i can't right and left clicking at my npc gui

### ISSUE THREAD

**Issue by Asukayuri1402** [samolego/TraderNPCs#6]
this is a problem video about the issue
im sorry my English is bad

this is the media fire video, its too big mb so yeah i used media fire link
https://www.mediafire.com/file/qwn2t367f66anbp/Minecraft__1.18.1_-_Singleplayer_2022-03-14_09-51-40.mp4/file

**Comment by samolego** [comment]
Can you describe the issue as well?
Also, could you upload the video to a streaming platform (yt, streamable, etc.)?

**Comment by Asukayuri1402** [comment]
ok

**Comment by Asukayuri1402** [comment]
here
https://www.youtube.com/watch?v=9sVIvuCDhrw

**Comment by samolego** [comment]
Using forge?

**Comment by Asukayuri1402** [comment]
yes

**Comment by Asukayuri1402** [comment]
i using forge

**Comment by samolego** [comment]
That's duplicate of #3 . I'm now waiting on https://github.com/architectury/architectury-loom/issues/75 to be resolved.

**Comment by Asukayuri1402** [comment]
so how to repair it?

**Comment by samolego** [comment]
Please read the above comment.
